crates/eip7702/src/auth_list.rs

```diff
@@ -348,16 +361,11 @@ mod tests {
     #[test]
     fn test_auth_json() {
         let sig = r#"{"r":"0xc569c92f176a3be1a6352dd5005bfc751dcb32f57623dd2a23693e64bf4447b0","s":"0x1a891b566d369e79b7a66eecab1e008831e22daa15f91a0a0cf4f9f28f47ee05","yParity":"0x1"}"#;
-        let auth = SignedAuthorization {
-            inner: Authorization {
-                chain_id: 1u64,
-                address: Address::left_padding_from(&[6]),
-                nonce: 1,
-            },
-            signature: serde_json::from_str(sig).unwrap(),
-        };
+        let auth =
+            Authorization { chain_id: 1u64, address: Address::left_padding_from(&[6]), nonce: 1 }
+                .into_signed(serde_json::from_str(sig).unwrap());
         let val = serde_json::to_string(&auth).unwrap();
-        let s = r#"{"chainId":"0x1","address":"0x0000000000000000000000000000000000000006","nonce":"0x1","r":"0xc569c92f176a3be1a6352dd5005bfc751dcb32f57623dd2a23693e64bf4447b0","s":"0x1a891b566d369e79b7a66eecab1e008831e22daa15f91a0a0cf4f9f28f47ee05","yParity":"0x1"}"#;
+        let s = r#"{"chainId":"0x1","address":"0x0000000000000000000000000000000000000006","nonce":"0x1","v":"0x1","r":"0xc569c92f176a3be1a6352dd5005bfc751dcb32f57623dd2a23693e64bf4447b0","s":"0x1a891b566d369e79b7a66eecab1e008831e22daa15f91a0a0cf4f9f28f47ee05"}"#;
```

Comment:
flagging that this would break rpc on our op-geth forks which currently operate on yParity

the field is also named yParity in the EIP so not sure if we should use v here

Reply:
geth uses `v` here and `yParity` can only contain bool so we are forced to use `v` here anyway
```
$ cast rpc 'eth_getTransactionByHash' '0x94b1254a5d3d1e9b620c45afdc503b7644861bfe95cad9adaa5872b6dfec2e6b' --rpc-url https://rpc.pectra-devnet-4.ethpandaops.io/ | jq
{
  "blockHash": "0x8a5db43806dedc59c886bfcebb70722b943fe0ba63e3942094d58c2ebd6ff11f",
  "blockNumber": "0x3347",
  "from": "0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266",
  "gas": "0x186a0",
  "gasPrice": "0x77359407",
  "maxFeePerGas": "0x7735940e",
  "maxPriorityFeePerGas": "0x77359400",
  "hash": "0x94b1254a5d3d1e9b620c45afdc503b7644861bfe95cad9adaa5872b6dfec2e6b",
  "input": "0x",
  "nonce": "0x0",
  "to": "0x0000000000000000000000000000000000000000",
  "transactionIndex": "0x3",
  "value": "0x0",
  "type": "0x4",
  "accessList": [],
  "chainId": "0x1a3ca344a",
  "authorizationList": [
    {
      "chainId": "0x1a3ca344a",
      "address": "0x14dc79964da2c08b23698b3d3cc7ca32193d9955",
      "nonce": "0x1",
      "v": "0x1",
      "r": "0xe6a5deff23b61ea8801f43623f138efec8f66ff757eb15b3c9f54d5c9f129316",
      "s": "0x6e7cd724e9530833632a6aa7595c69882f286217f62ac36fb0d84bc5ca8cc0d0"
    }
  ],
  "v": "0x0",
  "r": "0x11f7cbfb512894f202f1bcd1e5143efad07b61e11f6693f9ec5410afbcf2a4c0",
  "s": "0x13ae831aba6eb091ac686d6656918df3f3f24ae4284e48a0439edf10211fa322",
  "yParity": "0x0"
}
```

> flagging that this would break rpc on our op-geth forks which currently operate on yParity

not sure I understand which forks you mean though

Reply:
we should use yparity i guess